docs/docs/rest-api/public/api/v2/types/pod-status.raml

```diff
@@ -0,0 +1,58 @@
+#%RAML 1.0 DataType
+type: object
+description: |
+  A pod allows one to launch a collection co-located (on the same agent) containers
+  that share the same network namespace and that may share the same volumes.
+  Resources are specified on a per-container basis.
+additionalProperties: false
+properties:
+  id:
+    type: !include pathId.raml
+    description:|
+      Fully qualified name of the pod that this status object is describing.
+  spec:
```

Comment:
you want it to be optional? or you don't like the name? i'm open to naming suggetions